node-fs-readfile/cat.js

```diff
@@ -3,16 +3,17 @@ import { readFile } from 'node:fs/promises';
 const fileArray = process.argv.slice(2);
 
 async function print() {
-  await Promise.all(
-    fileArray.map(async (file) => {
-      try {
-        const data = await readFile(`./${file}`, { encoding: 'utf8' });
-        console.log(data);
-      } catch (error) {
-        console.log(error.message);
-      }
-    })
-  );
+  const array = await fileArray.map((file) => {
+    try {
+      const read = readFile(file, { encoding: 'utf8' });
```

Comment:
Since you don't `await` the `readFile` call, it will actually never throw an exception, so your catch won't ever get used.